TestCntlrApp/src/enbApp/nb_s1ap_hndlr.c

```diff
@@ -2456,6 +2456,7 @@ PUBLIC S16 nbPrcIncS1apMsg(NbUeCb *ueCb, S1apPdu *pdu, U8 msgType) {
     if (ret != ROK) {
       NB_LOG_ERROR(&nbCb, "Failed to handle S1 handover Cancel Ack");
     }
+      NB_LOG_ERROR(&nbCb, "Failed to handle S1 handover Cancel Ack");
```

Comment:
why is this added here?

Reply:
did you intend to add a DEBUG log that it was processed successfully?

Reply:
I think it got added by mistake while resolving merge conflicts